app/models/document.rb

```diff
@@ -9,7 +9,7 @@ class Document < ApplicationRecord
   STATES = %w[verified pending replaced rejected].freeze
 
   belongs_to :user
-  validates :doc_type, :doc_number, :upload, presence: true
+  validates :doc_type, :upload, presence: true
```

Comment:
doc_number is not passed for address documents usually, so we dont validate its presence here